docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,585 @@
+---
+title: EUID Client-Side Integration Guide for Mobile
+sidebar_label: Client-Side Integration for Mobile
+pagination_label: EUID Client-Side Integration Guide for Mobile
+description: Setting up a mobile integration with token generate and refresh both on the client side.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import EnableLogging from '/docs/snippets/_mobile-docs-enable-logging.mdx';
+import GMAIMA_Plugins from '/docs/snippets/_mobile_docs_gmaima-plugin-gss.mdx';
+import PrebidMobileSDK from '/docs/snippets/_mobile_docs_prebid-mobile.mdx';
+import ExampleAdvertisingToken from '/docs/snippets/_example-advertising-token.mdx';
+
+
+# EUID Client-Side Integration Guide for Mobile
+
+This guide is for mobile app publishers who want to integrate with EUID with changes only within their mobile app.
+
+These instructions do not apply to publishers who want to use a Private Operator, or who want to generate tokens server-side. Those publishers should follow the [Client-Server Integration Guide for Mobile](integration-mobile-client-server.md).
+
+This page provides a high-level overview of integration steps and links to additional documentation.
+
+EUID provides mobile SDKs for [Android](../sdks/sdk-ref-android.md) and [iOS](../sdks/sdk-ref-ios.md). Each SDK has the following features:
+
+- Generates an EUID <Link href="../ref-info/glossary-uid#gl-identity">identity</Link> (an EUID token and associated values) and persists it in local file storage.
+- Automatically refreshes EUID tokens.
+
+:::note
+This guide uses the group term **EUID mobile SDKs** to include both the SDK for Android and the SDK for iOS.
+:::
+
+For FAQs relating to mobile publisher integrations, see [FAQs for Mobile Integrations](integration-mobile-overview.md#faqs-for-mobile-integrations).
+
+To integrate with UIEUIDD2 client-side, you'll need to complete the following steps:
+
+1. [Complete the EUID account setup](#complete-the-euid-account-setup).
+
+1. [Add the EUID mobile SDK to your mobile app](#add-the-euid-mobile-sdk-to-your-mobile-app).
+
+1. [Configure the EUID mobile SDK](#configure-the-euid-mobile-sdk).
+
+1. [Check that the token was successfully generated and then pass it for bidstream use](#pass-generated-token-for-bidstream-use).
+
+1. [Optionally, integrate the EUID GMA/IMA Plugin for GAM Secure Signals integration](#optional-euid-gmaima-plugin-for-gam-secure-signals-integration).
+
+## Mobile SDK Version
+
+This guide provides instructions for using version 1.2.0 or higher of either of these EUID mobile SDKs:
+
+- SDK for Android
+- SDK for iOS
+
```

Comment:
Let's update the version. The IOS sdk version is 1.7.0 and android sdk version is 1.5.0

Reply:
@caroline-ttd : Done but a question: is it version number or higher, per the original copy, or does it have to be the exact version number? Original docs are inconsistent... other places it lists the specific version numbers, only.